docs/package.md

```diff
@@ -0,0 +1,91 @@
+# The Coalton `(package ...)` Form
+
+In Coalton, the `package` form is used to define a package by
+specifying which dependencies to import, and which definitions to
+export.
+
+The form also establishes the current package: any definitions that
+follow `(package <name>)` will be contained within that package.
+
+## Defining a Package
+
+A package in Coalton is defined using the `package` form. The syntax
+is:
+
+``` coalton
+(package <package-name>
+  (import <other-package-name>
+          (<other-package-name2 as <other2>)
+          ...)
+  (import-from <other-package-name>
+          <symbol1>
+          <symbol2>
+          ...)
+  (export <symbol3>
+          <symbol4>
+          ...))
+```
+
+An example:
```

Comment:
Maybe an example with all bells and whistles accounted for:

````
``` coalton
(package my-package
  (import coalton-library/list
          (coalton-library/vector as vec))
  (import-from coalton-library/char
          upcase
          downcase)
  (export sum-even-numbers))
```
````